src/MainWindow.vala

```diff
@@ -25,42 +25,53 @@ public class ShortcutOverlay.MainWindow : Gtk.Window {
     }
 
     construct {
-        var settings_button = new Gtk.Button () {
-            icon_name = "preferences-system-symbolic",
-            tooltip_text = _("Keyboard Settings…")
-        };
-
-        var shortcuts_view = new ShortcutsView () {
+        var box = new Gtk.Box (Gtk.Orientation.VERTICAL, 24) {
             margin_start = 36,
             margin_end = 36,
             margin_top = 12,
-            margin_bottom = 36
+            margin_bottom = 24
         };
 
-        child = shortcuts_view;
+        var shortcuts_view = new ShortcutsView ();
 
-        var start_controls = new Gtk.WindowControls (Gtk.PackType.START);
+        var settings_button = new Gtk.Button.with_label (_("Keyboard Settings…")) {
```

Comment:
Can we use a Gtk.LinkButton here instead? Would de-emphasize things in the UI a bit and actually be fewer lines to implement